### PR TITLE
Add getTimes inside SpikeReader::Population

### LINKED ISSUE #155: Query start and end time of spike reports
Hello,

Currently, the only way to know the start and end time of a spike report is by loading the whole report and search for the highest timestamp.

It would be useful to have a similar method as `ReportReader`'s `getTimes()` to directly return this value from the SpikeReader members.

### PR DESCRIPTION
**Pull request by alkino**
Return start and stop times.

Fix #155

### CODE REVIEW

**Review by NadirRoGue — APPROVED** [review]
Everything looks perfect

**Comment by mgeplf** [comment]
@alkino Before this goes in, can you add tests, and python bindings?

File: src/report_reader.cpp [hunk 1]
```diff
@@ -112,6 +112,10 @@ auto SpikeReader::openPopulation(const std::string& populationName) const -> con
     return populations_.at(populationName);
 }
 
+std::tuple<double, double> SpikeReader::Population::getTimes() const {
```
Comment (mgeplf) [inline src/report_reader.cpp:115]: can you add tests and python bindings for this?

**Review by mgeplf — CHANGES_REQUESTED** [review]

**Review by sergiorg-hpc — APPROVED** [review]

File: src/report_reader.cpp (hunk 1 above)
Comment (alkino) [reply src/report_reader.cpp:115]: done

**Review by mgeplf — APPROVED** [review]
Thanks for the tests!

**Pull request merged**